spec/basicAuth.spec.ts

```diff
@@ -0,0 +1,97 @@
+import { shallow, ShallowWrapper } from 'enzyme';
+import * as React from 'react';
+import BasicAuth from '../src/renderer/components/basic-auth';
+
+fdescribe('basic auth', () => {
+
+    it('should render correctly', () => {
+        const wrapper: ShallowWrapper = shallow(React.createElement(BasicAuth));
+
+        expect(wrapper).toMatchSnapshot();
+    });
+
+    it('should call `setState` when `change` is call', () => {
```

Comment:
typo. please change the statement to `should call setState when change is called`